modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppRestbedServerCodegen.java

```diff
@@ -100,6 +101,52 @@ public CppRestbedServerCodegen() {
         importMapping.put("restbed::Bytes", "#include <corvusoft/restbed/byte.hpp>");
     }
 
+    @Override
+    public Map<String, Object> updateAllModels(Map<String, Object> objs) {
+        // Index all CodegenModels by model name.
+        Map<String, CodegenModel> allModels = getAllModels(objs);
```

Comment:
Requires  #4753, to break method down into smaller pieces. Avoids just copy-pasting the same functionallity.